### DISCUSSION TITLE
Unable to build frontend for node 22.12

### DISCUSSION THREAD

**Discussion by onearthit** [open-webui/open-webui#7607] [Issues]
I have encounter this error during build

```
14.78 file:///app/tailwind.config.js:39
14.78 	plugins: [require('@tailwindcss/typography')]
14.78 	         ^
14.78 
14.78 ReferenceError: require is not defined
```

it was cause by `node:22-alpine3.20` in Dockerfile:24
I changed it into `node:22.11.0-alpine3.20` and it was fixed.

**Comment by kim-gtek** [comment]
I changed the file to tailwind.config.cjs and that fixed it for me as require works with .cjs file type

**Comment by mmcaulay** [comment]
I got the same error and tried both of your workarounds separately, both worked.
